crates/ty_python_semantic/src/types/class.rs

```diff
@@ -506,6 +523,14 @@ pub struct ClassLiteral<'db> {
     pub(crate) dataclass_transformer_params: Option<DataclassTransformerParams>,
 }
 
+impl<'db> ClassLiteral<'db> {
+    pub fn ordering(self, db: &'db dyn Db, other: Self) -> Ordering {
+        self.name(db)
+            .cmp(other.name(db))
+            .then_with(|| self.body_scope(db).cmp(&other.body_scope(db)))
```

Comment:
Nit: Doesn't this depend on a salsa id comparison?

Reply:
Not a nit! Thanks for catching that, I didn't realize `ScopeId` was interned as well.